src/ImageSharp/Formats/Jpg/Components/MutableSpan.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Buffers;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+
+namespace ImageSharp.Formats
+{
+    /// <summary>
+    /// Like corefxlab Span, but with an AddOffset() method for efficiency.
+    /// TODO: When Span will be official, consider replacing this class!
```

Comment:
By my understanding we would be looking at some perf improvements switching to span when it's out. Is that correct?